Packs/FeedIntel471/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Intel471 Feed",
     "description": "This content pack fetches actor and malware related indicators from Intel 471. It also fetches watcher alerts.",
     "support": "partner",
-    "currentVersion": "2.0.17",
+    "currentVersion": "2.0.18",
```

we should consider bumping it by minor in my opinion 
```suggestion
    "currentVersion": "2.1.0",
```

make sure to change the name of the RN also to `2_1_0.md`